node/modules/storageminer.go

```diff
@@ -1007,3 +1007,19 @@ func mutateCfg(r repo.LockedRepo, mutator func(*config.StorageMiner)) error {
 
 	return multierr.Combine(typeErr, setConfigErr)
 }
+
+func PopulateEnabledMinerSubsystems(cfg config.MinerSubsystemConfig) (res api.MinerSubsystems) {
```

Comment:
```suggestion
func EnabledMinerSubsystems(cfg config.MinerSubsystemConfig) (res api.MinerSubsystems) {
```

Populate implies "fill in" (i.e., in the config).